packs/feats/dragonblood-paragon.json

```
@@ -59,10 +59,22 @@
                 "key": "AdjustStrike",
                 "mode": "add",
                 "predicate": [
-                    "dragonbloodParagon:strongjawkobold"
+                    "dragonblood-paragon:strongjaw-kobold"
                 ],
                 "property": "weapon-traits",
                 "value": "deadly-d6"
+            },
+            {
+                "key": "Note",
+                "predicate": [
+                    "dragonblood-paragon:kobold-breath"
+                ],
+                "selector": [
+                    "kobold-breath-inline-damage",
+                    "dragons-breath-inline-damage"
+                ],
+                "text": "PF2E.SpecificRule.Kobold.DragonbloodParagon.KoboldBreath.Note",
+                "title": "Dragonblood Paragon"
```

```suggestion
                "title": "{item|name}"
```
Basically the idea is we never have displayed text like that in RE fields to make life easier for translation modules.